CONTRIBUTING.md

```diff
@@ -225,3 +225,68 @@ For building the copr locally, you will need:
 ```bash
 make copr-build
 ```
+
+### Avoiding calling yum API in some specific cases
+
+We found out a bug that happened during the handling of a SIGINT
+signal (or pressing CTRL + C), the problem was caused when a user pressed those
+combination of keys during any step of the conversion before the point of no
+return, thus, starting the process of rollback and failing in the first call
+`yum` call to check wether or not some packages were installed on the system.
+
+The problem is that when we enter in that state of rollback after a SIGINT,
+`librpm` is handling the signal on their code (without any possibility to
+change this) and when the librpm detects that a SIGINT was raised, it
+understands that the execution of process should stop and a exit code 1 is sent
+to the user. One way to bypass that is not calling `yum` methods that actually
+trigger `librpm` in the process, instead, prefer calling the `yum` binary or
+any other binary that achieve the same output you desire. The reason for this
+change is that `librpm` won't trigger their handling of SIGINT if we call the
+binary directly with `call_yum_cmd` or `run_subporcess` for example, as it
+understands it's a new process and that process wasn't in the middle of the
+SIGINT raising.
```

Comment:
```suggestion
The problem is that when we enter rollback after a SIGINT, `librpm` handles the
signal in its code (without any possibility to change this). The `librpm` handler
detects that a SIGINT was raised, decides that the execution of the process
should stop, and sends 1 as the program's exit code. One way to bypass this
problem is not to call `yum` methods that actually trigger `librpm` in the
process, instead, calling the `yum` binary or any other binary that achieves
the same goal. By calling `yum` via `call_yum_cmd()` or `run_subprocess()`,
a separate process is started and the SIGINT is handled by that separate
process rather than the main convert2rhel process.
```